pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -288,7 +295,7 @@ func (r *InferenceServiceReconciler) updateStatus(desiredService *v1beta1api.Inf
 		isReady := inferenceServiceReadiness(desiredService.Status)
 		if wasReady && !isReady { // Moved to NotReady State
```

Comment:
I think we should generate the event if wasReady && isReadyFalse to eliminate the false positive that generates the event for status unknown case.

Reply:
currently this block generate false event when the deployment is still in progress which marks the condition "unknown"